kit.yml

```diff
@@ -229,6 +271,10 @@ credentials:
       password: random 64
     autoscalerdb:
       password: random 64
+    cfnetworkingdb:
+      password: random 64
+    locketdb:
+      password: random 64
 
   autoscaler:
     autoscaler/servicebroker_account:
```

Comment:
Missing generating password for the service broker account.

Reply:
Taken care of on L282